crates/database/src/trades.rs

```diff
@@ -287,6 +288,7 @@ mod tests {
 
         let now = std::time::Instant::now();
         trades(&mut db, Some(&ByteArray([2u8; 20])), None)
+            .into_inner()
```

Comment:
Needed, because trades() now returns Instrumented which doesn't implement TryStreamExt, so we have to consume the Instrument (which drops the span) to get the wrapped type.